identity/migration/pom.xml

```diff
@@ -38,6 +38,11 @@
       <groupId>org.springframework</groupId>
       <artifactId>spring-context</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.18.0</version>
+    </dependency>
```

Comment:
:x: Please move dependency version management from `dependencies` to `dependencyManagament`